### PR TITLE
Adding some screens

### PR DESCRIPTION
**Pull request by erickzanardo**
Adds some screens to mock the app, and integrate the game with it

https://user-images.githubusercontent.com/835641/120829646-05ba5180-c534-11eb-9428-af7eff85a5fe.mov

### CODE REVIEW

File: lib/widgets/quick_action_card.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+              padding: const EdgeInsets.only(bottom: 10),
+              child: Icon(
+                icon,
+                color: Theme.of(context).accentColor,
+                size: 70,
+              ),
+            ),
+            Text(label, style: Theme.of(context).textTheme.bodyText2),
+          ],
+        ),),
```
Comment (luanpotter) [inline lib/widgets/quick_action_card.dart:31]: image

File: lib/widgets/arcade_call_to_action.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                        'Space burger arcade!',
+                        style: Theme.of(context).textTheme.headline5,
+                      ),
+                      SizedBox(height: 10),
+                      Text(
+                          'Proteja a terra contra invasores extraterrestres e ganhe prêmios!',
+                          textAlign: TextAlign.center,
+                      ),
+                    ],
+                  ),),
```
Comment (luanpotter) [inline lib/widgets/arcade_call_to_action.dart:38]: image

File: lib/game_page.dart [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+class _GamePageState extends State<GamePage> {
+  int _credits = 2;
+  bool _gameOver = false;
+  InvadersGame? _game;
+
+  void _play() {
+    setState(() {
+      _credits = _credits - 1;
```
Comment (luanpotter) [inline lib/game_page.dart:19]: `_credits--`?

File: lib/widgets/arcade_call_to_action.dart (hunk 2 above)
Comment (erickzanardo) [reply lib/widgets/arcade_call_to_action.dart]: Dramatic

File: lib/widgets/arcade_call_to_action.dart [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                Icon(
+                  Icons.videogame_asset_sharp,
+                  size: 100,
+                  color: Theme.of(context).primaryColorDark,
+                ),
+                Expanded(
+                  child: Padding(
+                      padding: const EdgeInsets.all(10),
+                      child: Column(
+                    children: [
```
[the remark this answers is not in the document]
Comment (erickzanardo) [reply lib/widgets/arcade_call_to_action.dart]: Dramatic

**Pull request merged**